recipes/r-hash/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: r-hash
+  version: "2.2.6"
+
+source:
+  fn: hash_2.2.6.tar.gz
+  url: http://cran.r-project.org/src/contrib/hash_2.2.6.tar.gz
```

Comment:
Would template out the version like in the [`example`]( https://github.com/conda-forge/staged-recipes/blob/bd4f4d9abd1cc664cf16a9f1fd615b299fe073eb/recipes/example/meta.yaml#L59-L63 ).